backend/app/models/__init__.py

```
@@ -62,3 +62,5 @@
     OpenAIConversationBase,
     OpenAIConversationCreate,
 )
+
+from .onboarding import OnboardingRequest, OnboardingResponse
```

Ruff (0.12.2)

`.onboarding.OnboardingRequest` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)